config/defaults.conf

```diff
@@ -95,8 +95,8 @@ reload_vocab = 0     // If true, force the rebuilding of the vocabulary files in
 is_probing_task = 0  // Set if the tasks in target_tasks are NLI-style probing tasks. (Ellie: Is this right?)
 
 // Learning curves
-training_data_fraction = 1  // Use only the specified fraction of the training data in the do_pretrain phase.
-                            // Should not impact eval-phase training, even for the same task.
+pretraining_data_fraction = 1  // Use only the specified fraction of the training data in the do_pretrain phase.
```

Comment:
Be consistent with part of speech: pretrain/finetune or pretraining/finetuning

Reply:
Actually, a suggestion: pretraining/target_training. The term _fine-tuning_ isn't ideal, since it _also_ refers to a separate decision about which parameters to update.

Reply:
Let's stick with target task then.